relay-server/src/endpoints/project_configs.rs

```diff
@@ -70,12 +71,17 @@ impl ProjectStateWrapper {
 ///
 /// Version 3 also adds a list of projects whose response is pending.  A [`ProjectKey`] should never
 /// be in both collections.  This list is always empty before V3.
+///
+/// The Response may also have a [`GlobalConfig`] which should be returned if the `global_config`
+/// flag is enabled on [`GetProjectStatesRequest`]
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
 struct GetProjectStatesResponseWrapper {
     configs: HashMap<ProjectKey, Option<ProjectStateWrapper>>,
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pending: Vec<ProjectKey>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    global: Option<GlobalConfig>,
```

Comment:
Can we stay consistent, please? Either call everything `gloabl_config` or `global`. 
```suggestion
    global_config: Option<GlobalConfig>,
```

Reply:
ok, I'll change it, added you as reviewer for the sentry side: https://github.com/getsentry/sentry/pull/53918